src/components/common/TooltipLegacy.tsx

```diff
@@ -0,0 +1,240 @@
+import React, { ReactNode, CSSProperties, useCallback } from 'react'
```

Comment:
Moved the legacy tooltip to the `components/common` folder as per @alfetopito 's suggestion.

Reply:
Why this file looks like a new file? surprised git didn't detect it as just a rename. And shouldn't be somewhere the deletion of the old file? I can't find it

Reply:
Because the old file name was Tooltip.tsx, and I use that name as our current Tooltip component. So I copied the previous Tooltip.tsx and renamed it to --> TooltipLegacy.tsx

Reply:
Actually, my suggestion is to move this further away, keeping the same name.
Instead of `src/components/common/TooltipLegacy`, I'd rather move it to `src/apps/gp-v1/components/common/Tooltip`.
And update `gp-v1` references to point to the same tooltip component but with a different path now.

Reply:
Ah I see your point, makes perfect sense. Will address in this PR.